package.json

```diff
@@ -166,6 +166,7 @@
     "@aws-cdk/aws-servicediscovery": "~1.172.0",
     "@aws-cdk/aws-sns": "~1.172.0",
     "@aws-cdk/aws-sqs": "~1.172.0",
-    "@aws-cdk/region-info": "~1.172.0"
+    "@aws-cdk/region-info": "~1.172.0",
+    "amplify-cli-core": "^3.5.1"
```

Comment:
I added this because I was getting type mismatch errors. Multiple versions of `amplify-cli-core` were being included.

```
src/storage/storage.ts:134:40 - error TS2345: Argument of type 'import("/Users/dppilche/amplify/amplify-category-api/node_modules/amplify-cli-core/lib/types").$TSContext' is not assignable to parameter of type 'import("/Users/dppilche/amplify/amplify-category-api/node_modules/amplify-category-function/node_modules/amplify-cli-core/lib/types").$TSContext'.
  The types of 'amplify.constructExeInfo' are incompatible between these types.
    Type '(context: import("/Users/dppilche/amplify/amplify-category-api/node_modules/amplify-cli-core/lib/types").$TSContext) => any' is not assignable to type '(context: import("/Users/dppilche/amplify/amplify-category-api/node_modules/amplify-category-function/node_modules/amplify-cli-core/lib/types").$TSContext) => any'.
      Types of parameters 'context' and 'context' are incompatible.
        Type 'import("/Users/dppilche/amplify/amplify-category-api/node_modules/amplify-category-function/node_modules/amplify-cli-core/lib/types").$TSContext' is not assignable to type 'import("/Users/dppilche/amplify/amplify-category-api/node_modules/amplify-cli-core/lib/types").$TSContext'.
          The types of 'amplify.showStatusTable' are incompatible between these types.
            Type '(resourceTableParams: import("/Users/dppilche/amplify/amplify-category-api/node_modules/amplify-category-function/node_modules/amplify-cli-core/lib/cliViewAPI").ViewResourceTableParams) => Promise<...>' is not assignable to type '(resourceTableParams: import("/Users/dppilche/amplify/amplify-category-api/node_modules/amplify-cli-core/lib/cliViewAPI").ViewResourceTableParams) => Promise<any>'.
              Types of parameters 'resourceTableParams' and 'resourceTableParams' are incompatible.
                Type 'import("/Users/dppilche/amplify/amplify-category-api/node_modules/amplify-cli-core/lib/cliViewAPI").ViewResourceTableParams' is not assignable to type 'import("/Users/dppilche/amplify/amplify-category-api/node_modules/amplify-category-function/node_modules/amplify-cli-core/lib/cliViewAPI").ViewResourceTableParams'.
                  Types have separate declarations of a private property '_command'.

134       const invoker = await getInvoker(context, { handler: config.handler, resourceName: triggerName, envVars: config.environment });
```

`/Users/dppilche/amplify/amplify-category-api/node_modules/amplify-cli-core` -> 3.4.0
`/Users/dppilche/amplify/amplify-category-api/node_modules/amplify-category-function/node_modules/amplify-cli-core` -> 3.5.1